.travis.yml

```diff
@@ -65,13 +65,6 @@ matrix:
            CONDA_DEPENDENCIES=""
            INSTALL_HOLE="false"
 
-    - env: NAME="python 2.7"
```

Comment:
Can't remember the exact issue but I think we said we would drop 3.5 too.

Reply:
I think it was this one: https://github.com/MDAnalysis/mdanalysis/issues/2541#issuecomment-590611719

Reply:
I *thought* dropping py2.7 and py3.5 would be a 2.0 thing, and that 1.xx releases would be just bug fixes, based on that same comment?

Reply:
Pinging @orbeckst here

Reply:
It looks like everyone else is moving on, I'd like to just ditch python 2.  If we find anything nasty that we fix I could do a backport/LTS thing, but I'd rather just force everyone to move to py 3, anaconda makes this possible enough.

Reply:
I was about to tag that comment! ;) I think this discussion is suited for the dev list, but if major packages are already dropping support for 3.5 (see https://github.com/MDAnalysis/mdanalysis/issues/2541#issuecomment-589831731) probably we should drop it too.

Reply:
> I thought dropping py2.7 and py3.5 would be a 2.0 thing, and that 1.xx releases would be just bug fixes, based on that same comment?

My view here (probably to be discussed on the dev list) is that it would be nice to increase the number of releases to include small bug fixes. The reason is that many issues are solved relatively quickly, but make it on master after a very long time (and therefore most users don't benefit from them).

I think we should start dropping Python 2 and 3.5 support with 1.0.x releases (as done here) and tidy up the code as much as possible on the way to 2.0 (possibly with type hints, and other cool stuff).

Reply:
In https://www.mdanalysis.org/2019/11/06/roadmap/ we said _1.x will be the last version of MDAnalysis that fully supports Python 2.7_

This means that 1.x supports and tests 2.7 so we cannot remove Python 2.7 tests. Drop it for 2.0.

I'll start a discussion on the dev list instead of an issue thread...